GSA-TTS/FAC
# 3705 address historical federal awards with extraneous passthrough names

## Description
One of the current validation rules for federal awards stipulates that when an award is categorized as a direct award (i.e., when the `is_direct` flag is set to `Y`), then the passthrough entity fields (name and ID) must be blank. Conversely, when an award is categorized as an indirect award (is_direct flag set to `N`), the passthrough entity name must be provided. Some historical reports have awards categorized as direct, but the passthrough entity fields associated with these reports are not blank, which causes them to fail validation. This PR addresses the issue by replacing the `is_direct` flag with `GSA_MIGRATION` to indicate that the provided value is incorrect and allowing the report to pass validation.
Associated ticket is #3705.

## How to test
1. Launch the stack from the main branch.
2. Identify a valid report with passthrough information.
3. Change any `ELECAUDITS.DIRECT` field associated with this report from `N` to `Y`.
4. Attempt to migrate this report and notice how it fails due to "When direct award is `Y`, no passthrough name is required."
5. Switch to this PR branch and re-run the migration for the same report and observe that it processes properly.
6. Check the `FederalAward.is_direct` field (dissemination table) for the modified flag and confirm that it has been replaced with `GSA_MIGRATION`.
7. For extra points, you may try to create from scratch an audit report and submit it from this PR branch to confirm that process works as expected.

## PR checklist: submitters

## PR checklist: reviewers


The larger the PR, the stricter we should be about these points.
